includes/elements/section.php

```diff
@@ -1528,9 +1528,13 @@ private function print_shape_divider( $side ) {
 		$settings = $this->get_active_settings();
 		$base_setting_key = "shape_divider_$side";
 		$negative = ! empty( $settings[ $base_setting_key . '_negative' ] );
+		$shape_path = Shapes::get_shape_path( $settings[ $base_setting_key ], $negative );
+		if ( ! file_exists( $shape_path ) ) {
```

Comment:
Check also it's not a URL.

```suggestion
		if ( ! is_file(  $shape_path ) ||  ! file_exists( $shape_path ) ) {
```